### PR TITLE
SUS-3495 Replaced events with rollup

### PR DESCRIPTION
**Pull request by jcellary**
https://wikia-inc.atlassian.net/browse/SUS-3495

Replaced usage of events table with rollup

### CODE REVIEW

File: extensions/wikia/FounderEmails/FounderEmails.class.php [hunk 1]
```diff
@@ -215,23 +215,20 @@ public function getPageViews ( $cityID ) {
...
-			$db = wfGetDB( DB_SLAVE, array(), $wgStatsDB );
+		$db = wfGetDB( DB_SLAVE, array(), $wgDWStatsDB );
 
-			$oRow = $db->selectRow(
-				array( 'events' ),
-				array( 'count(0) as cnt' ),
-				array(  " rev_timestamp between '$today 00:00:00' and '$today 23:59:59' ", 'wiki_id' => $cityID ),
-				__METHOD__
-			);
+		$oRow = $db->selectRow(
```
Comment (mszabo-wikia) [inline extensions/wikia/FounderEmails/FounderEmails.class.php]: We can use `$db->selectField` helper here

**Review by mszabo-wikia — CHANGES_REQUESTED** [review]

File: extensions/wikia/FounderEmails/FounderEmails.class.php (hunk 1 above)
Comment (jcellary) [reply extensions/wikia/FounderEmails/FounderEmails.class.php]: :+1:

**Comment by jcellary** [comment]
@TK-999 I made the fix, please let me know if it's good now.

**Review by mszabo-wikia — APPROVED** [review]

File: extensions/wikia/FounderEmails/FounderEmails.class.php [hunk 2]
```diff
@@ -215,25 +215,18 @@ public function getPageViews ( $cityID ) {
 	}
 
 	public function getDailyEdits ( $cityID, /*Y-m-d*/ $day = null ) {
-		global $wgStatsDB, $wgStatsDBEnabled;
+		global $wgDWStatsDB;
 
-		$edits = 0;
-		if ( !empty( $wgStatsDBEnabled ) ) {
-			$today = ( empty( $day ) ) ? date( 'Y-m-d', strtotime( '-1 day' ) ) : $day;
+		$today = ( empty( $day ) ) ? date( 'Y-m-d', strtotime( '-1 day' ) ) : $day;
```
Comment (macbre) [inline extensions/wikia/FounderEmails/FounderEmails.class.php:220]: It's rather `$yesterday` :)

**Review by macbre — APPROVED** [review]

**Review by mszabo-wikia — APPROVED** [review]

**Review by macbre — APPROVED** [review]

**Pull request merged**